src/styles/blocks/goodsCard.scss

```
@@ -0,0 +1,74 @@
+.goodsCard {
+  @include goodsCard;
+  padding: 0 16px 16px;
+  width: 200px;
+  height: 408px;
+
+  &__image {
+    margin: 32px 3px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    height: 36px;
+    width: 166px;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $accent-text-color;
+    margin: 40px 0 0;
+  }
+
+  &__article {
+    font-size: 10px;
+    line-height: 14px;
+    color: $basic-text-color;
+    margin: 4px 0 0;
+  }
+
+  &__rating {
+    @include inlineBlock;
+    width: 166px;
+    height: 16px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 16px 0 0;
+  }
+
+  &__price {
+    @include inlineBlock;
+    width: 166px;
+    height: 18px;
+    margin: 24px 0 0;
+  }
+
+  &__note {
+    margin: 0;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 400;
+  }
+
+  &__value {
+    margin: 0;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    @include cardButton;
+    margin: 16px 0 0;
+
+    &[data-qa="hover"]:hover {
```

```suggestion
    &:hover {
```
Don`t increase the specificity of the selector